packages/devtools/lib/src/tables.dart

```diff
@@ -46,8 +46,15 @@ class Table<T> extends Object with SetStateMixin {
   int get rowCount => data?.length ?? 0;
 
   Column<T> _sortColumn;
+
   SortOrder _sortDirection;
 
+  set sortColumn(Column<T> column) {
```

Comment:
what if you don't want to sort any column? Seems reasonable for this to sometimes be null.

Reply:
If you don't want a sort column then you would never call this setter and _sortColumn would remain null.